### DISCUSSION TITLE
Update within SystenInformer with error 381 in DLL

### DISCUSSION THREAD

**Discussion by petesky** [winsiderss/systeminformer#1903] [Questions / Answers] [answered]
SystemInformer.exe - Ordnungszahl nicht gefunden: Die Ordnungszahl 381 wurde in der DLL "C:\Program Files\SystemInformer\SystemInformer.exe" nicht gefunden. A fresh install works on W11 x64. Whats wrong ?

**Comment by jxy-s** [comment, marked as answer]
Should be fixed with the following:
https://github.com/winsiderss/systeminformer/commit/70261ec6a31e6503c69029b033382d2f72b286c1
https://github.com/winsiderss/systeminformer/commit/28e4c066a22863c46ecb0cf1a898961e873031a6
https://github.com/winsiderss/systeminformer/commit/04ece53c3676b55ae77f7412788f3d976b125d0b

Will be in next build 👍

**Comment by petesky** [comment]
Still same problem ? Not fixed ?
